lerobot/common/datasets/push_dataset_to_hub/_download_raw.py

```diff
@@ -70,6 +70,68 @@
+    "lerobot-raw/utokyo_xarm_pick_and_place_converted_externally_to_rlds": "openx_rlds.utokyo_xarm_pick_and_place_converted_externally_to_rlds",
+    "lerobot-raw/utokyo_xarm_bimanual_converted_externally_to_rlds": "openx_rlds.utokyo_xarm_bimanual_converted_externally_to_rlds",
+    "lerobot-raw/robo_net": "openx_rlds.robo_net",
+    "lerobot-raw/robo_set": "openx_rlds.robo_set",
+    "lerobot-raw/berkeley_mvp_converted_externally_to_rlds": "openx_rlds.berkeley_mvp_converted_externally_to_rlds",
+    "lerobot-raw/berkeley_rpt_converted_externally_to_rlds": "openx_rlds.berkeley_rpt_converted_externally_to_rlds",
+    "lerobot-raw/kaist_nonprehensile_converted_externally_to_rlds": "openx_rlds.kaist_nonprehensile_converted_externally_to_rlds",
+    "lerobot-raw/stanford_mask_vit_converted_externally_to_rlds": "openx_rlds.stanford_mask_vit_converted_externally_to_rlds",
+    "lerobot-raw/tokyo_u_lsmo_converted_externally_to_rlds": "openx_rlds.tokyo_u_lsmo_converted_externally_to_rlds",
+    "lerobot-raw/dlr_sara_pour_converted_externally_to_rlds": "openx_rlds.dlr_sara_pour_converted_externally_to_rlds",
+    "lerobot-raw/dlr_sara_grid_clamp_converted_externally_to_rlds": "openx_rlds.dlr_sara_grid_clamp_converted_externally_to_rlds",
+    "lerobot-raw/dlr_edan_shared_control_converted_externally_to_rlds": "openx_rlds.dlr_edan_shared_control_converted_externally_to_rlds",
+    "lerobot-raw/asu_table_top_converted_externally_to_rlds": "openx_rlds.asu_table_top_converted_externally_to_rlds",
+    "lerobot-raw/stanford_robocook_converted_externally_to_rlds": "openx_rlds.stanford_robocook_converted_externally_to_rlds",
+    "lerobot-raw/imperialcollege_sawyer_wrist_cam": "openx_rlds.imperialcollege_sawyer_wrist_cam",
+    "lerobot-raw/iamlab_cmu_pickup_insert_converted_externally_to_rlds": "openx_rlds.iamlab_cmu_pickup_insert_converted_externally_to_rlds",
+    "lerobot-raw/uiuc_d3field": "openx_rlds.uiuc_d3field",
+    "lerobot-raw/utaustin_mutex": "openx_rlds.utaustin_mutex",
+    "lerobot-raw/berkeley_fanuc_manipulation": "openx_rlds.berkeley_fanuc_manipulation",
+    "lerobot-raw/cmu_playing_with_food": "openx_rlds.cmu_playing_with_food",
+    "lerobot-raw/cmu_play_fusion": "openx_rlds.cmu_play_fusion",
+    "lerobot-raw/cmu_stretch": "openx_rlds.cmu_stretch",
+    "lerobot-raw/berkeley_gnm_recon": "openx_rlds.berkeley_gnm_recon",
+    "lerobot-raw/berkeley_gnm_cory_hall": "openx_rlds.berkeley_gnm_cory_hall",
+    "lerobot-raw/berkeley_gnm_sac_son": "openx_rlds.berkeley_gnm_sac_son",
+    "lerobot-raw/droid": "openx_rlds.droid",
+    "lerobot-raw/droid100": "openx_rlds.droid100",
+    "lerobot-raw/fmb": "openx_rlds.fmb",
+    "lerobot-raw/dobbe": "openx_rlds.dobbe",
+    "lerobot-raw/usc_cloth_sim_converted_externally_to_rlds": "openx_rlds.usc_cloth_sim_converted_externally_to_rlds",
+    "lerobot-raw/plex_robosuite": "openx_rlds.plex_robosuite",
+    "lerobot-raw/conq_hose_manipulation": "openx_rlds.conq_hose_manipulation",
```

Comment:
I have 2 questions about this:
- Did you merge some raw datasets together to produce non-raw datasets? If so, is there a table somewhere I can use to know which LeRobot dataset (in `lerobot.available_datasets`) were create from which of these raw datasets? I'm seeing 62 new raw datasets here while there's only 40 new datasets in `lerobot.available_datasets`.
- If not, we should keep the naming convention we previously used: `lerobot/some_dataset` should be coming from `lerobot-raw/some_dataset_raw`. If you do this, no need to reupload everything as I believe you can simply do something like this:
```python
from huggingface_hub import HfApi

old_new_ids = {
    "lerobot-raw/some_old_name": "lerobot-raw/some_new_name",
    "lerobot-raw/some_other_old_name": "lerobot-raw/some_other_new_name",
}

api = HfApi()
for old_name, new_name in old_new_ids.items():
    api.move_repo(from_id=old_name, to_id=new_name, repo_type="dataset")
```